src/rpcblockchain.cpp

```diff
@@ -1422,34 +1241,6 @@ UniValue magnitude(const UniValue& params, bool fHelp)
     return results;
 }
 
-UniValue mymagnitude(const UniValue& params, bool fHelp)
-{
```

Comment:
We should make `magnitude` default to the user's current magnitude now.

Reply:
so rename that to magnitude and simply display the magnitude of the user (current user in msPrimaryCPID?) 
and cpid

Reply:
We can reuse the current `magnitude` :)

Reply:
lost train on thought of where this one was to be changed since mymagnitude is removed

Reply:
added it to magnitude report. maybe that will satisfy instead of re adding this rpc

Reply:
I was thinking we change `magnitude` to default to the user's CPID:


Change
```
std::string cpid;
if (params.size() > 0)
    cpid = params[0].get_str();
```

to
```
const std::string cpid = !params.empty()
        ? params[0]
        : msPrimaryCPID;

if(cpid.empty())
   throw... something
```

Reply:
oh so revive magnitude to have this information then

Reply:
`magnitude` was never axed. We have `magnitude` which allows you to specify a CPID, and `mymagnitude` which is exactly the same but works on `msPrimaryCPID` (?). We can merge this to make `magniude` work on any specified CPID, or fall back to `msPrimaryCPID` if none is specified.